src/literals.py

```diff
@@ -21,6 +21,9 @@
 CLIENT_PORT = 2379
 PEER_PORT = 2380
 
+INTERNAL_USER = "root"
```

Comment:
is it possible to create `operator` user with `root` role instead of `root` user?

Reply:
Not instead, only additionally. The `root` user always has to be created in etcd (see [here](https://etcd.io/docs/v3.5/op-guide/authentication/rbac/#special-users-and-roles)). We could only have the `operator` user in addition to `root`, which doesn't really make sense from my point of view.

Reply:
lets than use `root` word instead of `admin` everywhere

Reply:
Yes we do, with this literal `INTERNAL_USER` it is transparent in the code, no hardcoded usernames or so.